File: _api/Tada.API/Models/Message.cs
using System;

namespace Tada.API.Models
{
    public class Message
    {
        public int Id { get; set; }
        public string Content { get; set; }
        public bool IsRead { get; set; }
        public DateTime SentDate { get; set; }
        public DateTime? ReadDate { get; set; }
        public int SenderId { get; set; }
        public int ReceiverId { get; set; }
        public bool SenderDeleted { get; set; }
        public bool ReceiverDeleted { get; set; }
        public User Sender { get; set; }
        public User Receiver { get; set; }
    }
}

File: _client/Tada-SPA/src/app/models/message.ts
export interface Message {
    id: number;
    content: string;
    isRead: boolean;
    sentDate: string;
    readDate: string | null;
    senderId: number;
    receiverId: number;
    senderDeleted: boolean;
    receiverDeleted: boolean;
    senderKnownAs: string;
    senderPhotoUrl: string;
    receiverKnownAs: string;
    receiverPhotoUrl: string;
}
